primitives/manta/src/assets.rs

```diff
@@ -144,48 +216,35 @@ impl Default for AssetRegistrarMetadata {
             decimals: 12,
             evm_address: None,
             is_frozen: false,
-            min_balance: DEFAULT_ASSET_ED,
+            min_balance: TEST_DEFAULT_ASSET_ED,
             is_sufficient: true,
         }
     }
 }
+*/
 
-impl AssetMetadata for AssetRegistrarMetadata {
-    fn min_balance(&self) -> Balance {
-        self.min_balance
+impl<B> BalanceType for AssetRegistryMetadata<B> {
+    type Balance = B;
+}
+
+impl<B> AssetMetadata for AssetRegistryMetadata<B> {
+    #[inline]
+    fn min_balance(&self) -> &B {
+        &self.min_balance
     }
 
+    #[inline]
```

Comment:
I'm going to leave this unresolved comment for myself, to run the benchmarks with/without all the inlines. Curious how much it actually changes the perf.